### PR TITLE
docs: update Android docs for new features

### PR DESCRIPTION
**Pull request by space-pope**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [X] I have run `npm test` against my changes and tests pass.
- [X] I have added or edited necessary documentation, or no docs changes are needed.

### Description

This updates the Android quickstart guide to reflect the latest version number and new features and the ASR guide to include the Spokestack cloud ASR.

### CODE REVIEW

File: content/docs/Concepts/asr.md [hunk 1]
```diff
@@ -65,6 +66,21 @@ None required! &#x1F389;
...
+Spokestack's Cloud ASR requires requests to be signed with a Spokestack client ID and API secret. Spokestack accounts are free, and cloud-based ASR currently is as well. If you don't already have an account, you can sign up for one [here](/create); if you do, [log in](/login) to get your credentials.
+
+##### Android
+
+- `spokestack-id` (string): A Spokestack client ID, available in the account portal.
+- `spokestack-secret` (string): A Spokestack API secret, also available in the account portal.
+
+##### iOS
+
+Same as the Android requirements.
```
Comment (timmywil) [inline content/docs/Concepts/asr.md]: I recommend repeating here instead of referring somewhere else, even if it's right above it.

Comment (space-pope) [reply content/docs/Concepts/asr.md]: Fair point. I waffled on that a bit myself.

File: content/docs/Android/getting-started.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ dependencies {
 
 // a new top-level block if you don't already have native dependencies
 native_dependencies {
-    artifact 'io.spokestack:spokestack-android:5.6.0'
+    artifact 'io.spokestack:spokestack-android:7.0.0'
```
Comment (timmywil) [inline content/docs/Android/getting-started.md]: I forgot about this PR, but might as well update the versions here before merging.

**Comment by space-pope** [comment]
Thanks for catching that. I think I might have a bit more I can remove from the getting started guide, but I have to do some experimentation, so might as well make that a separate thing.

**Review by timmywil — APPROVED** [review]

**Pull request merged**